bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/communication/RotelConnector.java

```diff
@@ -136,6 +136,9 @@
     public static final String KEY_DSP_MODE = "dsp_mode";
     public static final String KEY_DIMMER = "dimmer";
     public static final String KEY_FREQ = "freq";
+    public static final String KEY_BYPASSx = "bypass";
```

Comment:
Please do not use lowercase letter in a constant.

Reply:
And is this final x really expected or is it a typo?

Reply:
in your original version, the term BYPASS already appears; but I did not understand what function it covered. Therefore, I have called "my" BYPASS BYPASSx to distinguish it from others.

In order to remove the lowercase "x" I follow the description in Rotel's user manual where this funtionality is titled "Tone Control Bypass"; so I use instead of BYPASSx the string TCBYPASS (Tone Control Bypass)